packages/neo-one-website/src/components/docs/DocFooter.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { Box, Grid, styled } from 'reakit';
+import { prop } from 'styled-tools';
+import { AdjacentInfo } from '../../utils';
+import { AdjacentLink } from './AdjacentLink';
+
+interface Props {
+  readonly next?: AdjacentInfo;
+  readonly previous?: AdjacentInfo;
+}
+
+const Footer = styled(Grid)`
+  grid-template: 'prev next';
```

Comment:
I would simplify this to just `grid-auto-flow: column` and remove the grid areas.